### ISSUE TITLE
main file router.js uses src not dist

### ISSUE THREAD

**Issue by stukennedy** [gabrielmoreira/riot-router#34]
when building riot-router we get an error in Uglify as it doesn't recognise `class Router` (being es2015)
It seems that you are requiring the src instead of the dist in your main package file.

```js
var riot = require('riot');
var route = require('riot-route');
var Router = require('./src/router.js');
var router = Router.create({route: route});
if (window) {
    window.router = router;
}
module.exports = Router;
```
shouldn't line 3 be:
```js
var Router = require('./dist/router.js');
```
?

**Comment by gabrielmoreira** [comment]
@continuata Thank you.

I've released Riot-Router 0.9.7 using main file "dist/router.min.js".

Can you test again?

**Comment by stukennedy** [comment]
sorry for not responding before now, yes that all works great now.
